posts/bayesian-model-comparison/index.qmd

```diff
@@ -417,62 +426,78 @@ However if we can specify costs then decision making based on these would be the
 In our situation, in order to do cost-sensitive analysis/decision making, all we need is to specify a cost function -- we won't need to run an optimisation on this objective function. 
```

Comment:
The second paragraph of this section (too high up for me to comment) I think could maybe be improved.